src/ontology/external/mondo-omim-genes.robot.tsv

```diff
@@ -44,7 +44,6 @@ http://purl.obolibrary.org/obo/MONDO_0007163	http://identifiers.org/hgnc/1388	OM
 http://purl.obolibrary.org/obo/MONDO_0007164	http://identifiers.org/hgnc/12642	OMIM:108600	https://omim.org/entry/185880
 http://purl.obolibrary.org/obo/MONDO_0007167	http://identifiers.org/hgnc/3755	OMIM:108720	https://omim.org/entry/603381
 http://purl.obolibrary.org/obo/MONDO_0007168	http://identifiers.org/hgnc/3755	OMIM:108721	https://omim.org/entry/603381
-http://purl.obolibrary.org/obo/MONDO_0007171	http://identifiers.org/hgnc/4279	OMIM:108770	https://omim.org/entry/121013
```

Comment:
NOTE: This should not have been removed. See https://github.com/monarch-initiative/omim/blob/main/data/protected-disease-gene.tsv#L6

Reply:
This is fixed now by Joe quickly getting a fix in place in the OMIM repo for this issue and then running the goal `external/mondo-omim-genes.robot.tsv` in this data build branch to get this latest file.